src/Components/Home/Users.js

```diff
@@ -32,8 +32,8 @@ function Users({ list }) {
         }
       />
       <div className="user-list p-d-flex p-flex-wrap p-jc-center">
-        {!!filteredList &&
-          filteredList.length > 0 &&
+        {
+          filteredList?.length > 0 &&
```

Comment:
This is not related to this PR, it should go in another PR. But I am not sure it is needed, from the React docs they recommend keeping it the previous style https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator

Reply:
Okay I'll keep it in the previous style